.gitignore

```diff
@@ -1,7 +1,3 @@
 /node_modules/
 /typings/
-/jspm_packages/
-/dist/
-npm-debug.log
-.sw*
-.idea
```

Comment:
I think we should keep all these things in gitignore, minus maybe the `jspm_packages` one - `.sw*` is for Vim swap files, `.idea` is for IntelliJ, and `npm-debug.log` is for npm erroring out, all valid things to ignore.